addons/rangecard/functions/fnc_updateRangeCard.sqf

```diff
@@ -107,7 +107,10 @@ private _transonicStabilityCoef = _ammoConfig select 4;
 private _dragModel = _ammoConfig select 5;
 private _atmosphereModel = _ammoConfig select 8;
 
-private _useABConfig = (missionNamespace getVariable [QEGVAR(advanced_ballistics,enabled), false]);
+private _useABConfig = (
```

Comment:
really don't think this is correct
`_useABConfig` is used in many other places that have nothing to do with `barrelLengthInfluenceEnabled`

Reply:
> really don't think this is correct

Upon further inspection I also agree, this isn't correct.

---
What irritates me is that `_useABConfig` was defined, but there are 6 other instances of `missionNamespace getVariable [QEGVAR(advanced_ballistics,enabled),false]` being used instead of `_useABConfig`.